benchmark/kmeans.py

```diff
@@ -210,7 +209,7 @@ def benchmark_data_initialization(
                 f" {sample_weight} instead."
             )
 
-        return X, sample_weight, init
+        return X, sample_weight, init, n_clusters
 
     kmeans_timer = KMeansLloydTimeit(
```

Comment:
I have tried to run the benchmark against this branch and I got:

```python-traceback
Traceback (most recent call last):
  File "/io/code/sklearn-numba-dpex/benchmark/kmeans.py", line 214, in <module>
    kmeans_timer = KMeansLloydTimeit(
  File "/io/code/sklearn-numba-dpex/benchmark/kmeans.py", line 47, in __init__
    ) = data_initialization_fn(**data_initialization_kwargs)
TypeError: benchmark_data_initialization() missing 1 required positional argument: 'init'
```

I did not try to debug yet.

Reply:
Oops, easy fix fortunately